make.py

```diff
@@ -229,7 +230,7 @@ def htmlhelp():
           f".hhp project file in {FULL_BUILD_PREFIX}/htmlhelp.")
 
 
-@app.command()
+@app.command(rich_help_panel="Docs")
 def qthelp():
```

Comment:
For the less standard help commands such as this one, is there group these under `Additional Help Formats` or something similar?